app/routes.js

```diff
@@ -0,0 +1,24 @@
+var	express = require('express');
+var	config = require('./controller/settings');
+
+var	router = express.Router();
```

Comment:
Not sure you even need this here as long as you include it in app before you include the routes file?